geth/node/manager.go

```diff
@@ -69,14 +77,108 @@ func (m *NodeManager) startNode(config *params.NodeConfig) (<-chan struct{}, err
 
 	m.initLog(config)
 
-	ethNode, err := MakeNode(config)
+	deliveryManager := new(notifications.DeliveryService)
```

Comment:
Different notation like `var delivery notifications.DeliveryService` and `&deliveryManager` above. Would like to see it consistent.